table/table.go

```diff
@@ -27,6 +27,18 @@ import (
 	"github.com/pingcap/tidb/util/types"
 )
 
+// Type , the type of table, store data in different ways.
```

Comment:
Why add a space between `Types` and `,`?

Reply:
this is the rule of golint, otherwise it will report like this, "comment on exported type Type should be of the form "Type ..." (with optional leading article)"